$NetBSD: patch-ac,v 1.4 2009/01/05 12:15:17 he Exp $

This corresponds to http://rt.cpan.org/Public/Bug/Display.html?id=41060
Can't call method "isa" without a package or object reference.

Also correct a grammar error (duplication).

--- lib/ExtUtils/MakeMaker.pm.orig	2008-10-20 20:18:35.000000000 +0200
+++ lib/ExtUtils/MakeMaker.pm
@@ -122,7 +122,7 @@ sub _verify_att {
 
         my @sigs   = ref $sig ? @$sig : $sig;
         my $given  = ref $val;
-        unless( grep { $given eq $_ || ($_ && eval{$val->isa($_)}) } @sigs ) {
+        unless( grep { $given eq $_ || ($_ && eval{UNIVERSAL::isa($val,$_)}) } @sigs ) {
             my $takes = join " or ", map { _format_att($_) } @sigs;
 
             my $has = _format_att($given);
@@ -1708,7 +1708,7 @@ If set to 'none', no man pages will be i
 =item INSTALLVENDORSCRIPT
 
 Used by 'make install' which copies files from INST_SCRIPT to this
-directory if INSTALLDIRS is set to is set to vendor.
+directory if INSTALLDIRS is set to vendor.
 
 =item INST_ARCHLIB
 
